src/API/Logs/Bridge.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Logs;
+
+use OpenTelemetry\API\Common\Instrumentation\Globals;
+
+/**
+ * OpenTelemetry logs bridge API. To be used when implementing a handler/appender for existing logging libraries.
+ * This API is not meant to be used by application developers, but rather by library developers for the purpose
+ * of integrating existing logging libraries with OpenTelemetry.
+ *
+ * @see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md
+ */
+class Bridge
```

Comment:
What is the purpose of this class / why should `new Bridge('scope', $loggerProvider)` be used instead of `$loggerProvider->getLogger('scope')`?
> The Logs Bridge API consist of these main classes:
    [LoggerProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#loggerprovider) is the entry point of the API. It provides access to Loggers.
    [Logger](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#logger) is the class responsible for emitting logs as [LogRecords](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#logrecord).

> The log appender implementation will typically acquire a [Logger](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#logger) from the global [LoggerProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#loggerprovider) at startup time, then call [Emit LogRecord](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md#emit-logrecord) for LogRecords received from the application.

Reply:
It's a good question - I am struggling to work out what the bridge API "is". Is it a class (like I've implemented here), or an interface for log appenders to implement, or is it even looser than that: some stated criteria that a log appender should conform to?
The second quote there hints that it's the third option; a log appender satisfies bring a bridge API if it does the things that the spec says. I don't think we can enforce that via interfaces, perhaps as an abstract class?

Reply:
I've updated the PR to remove Bridge, and added a small README describing bridge/log-appender expectations.